src/QsCompiler/SyntaxProcessor/ExpressionVerification.fs

```diff
@@ -753,8 +753,8 @@ type QsExpression with
                 |> inference.Constrain
                 |> List.iter diagnose
             else
-                let diagnostics =
-                    inference.Constrain(callable.ResolvedType <. ResolvedType.New(QsTypeKind.Function(argType, output)))
+                let functionType = ResolvedType.withKind (QsTypeKind.Function(argType, output)) callable.ResolvedType
+                let diagnostics = inference.Constrain(callable.ResolvedType <. functionType)
```

Comment:
This might be a silly question, but couldn't find the answer in F#'s documentation. What is the `<.` operator doing here?

Reply:
It's a custom operator that we define:

https://github.com/microsoft/qsharp-compiler/blob/d21f8fa62eedbc3e1c2850f004350f0c3f67a8ca/src/QsCompiler/SyntaxProcessor/TypeInference/Constraint.fsi#L82-L85